scripts/run_large_benchmark.sh

```
@@ -0,0 +1,424 @@
+export NEO4J_USER
+export MEMGRAPH_USER
+
+if [[ "${RUN_NEO4J}" == "1" ]]; then
+  echo "==> Clearing Neo4j database (neo4j)"
+  echo "  - Stopping Neo4j to remove persistent data (large dataset requires fresh start)"
+  neo4j stop >/dev/null 2>&1
+  
+  echo "  - Removing Neo4j data directory"
+  NEO4J_DATA_DIR="/opt/homebrew/var/neo4j/data"
+  if [[ -d "$NEO4J_DATA_DIR" ]]; then
+    rm -rf "$NEO4J_DATA_DIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)